lib/widget/helpers/widgets.dart

```diff
@@ -56,48 +56,56 @@ class BoxWrapper extends StatelessWidget {
-                boxShadow: !boxController.hasBoxShadow()
-                    ? null
-                    : <BoxShadow>[
-                        BoxShadow(
-                            color: boxController.shadowColor ??
-                                ThemeManager().getShadowColor(context),
-                            blurRadius:
-                                boxController.shadowRadius?.toDouble() ??
-                                    ThemeManager().getShadowRadius(context),
-                            offset: boxController.shadowOffset ??
-                                ThemeManager().getShadowOffset(context),
-                            blurStyle: boxController.shadowStyle ??
-                                ThemeManager().getShadowStyle(context))
-                      ]));
+      width: ignoresDimension ? null : boxController.width?.toDouble(),
+      height: ignoresDimension ? null : boxController.height?.toDouble(),
+      margin: ignoresMargin ? null : boxController.margin,
+      padding: ignoresPadding ? null : boxController.padding,
+      clipBehavior: clip,
+      decoration: !boxController.hasBoxDecoration()
+          ? null
+          : BoxDecoration(
+              color: boxController.backgroundColor,
+              gradient: boxController.backgroundGradient,
+              border: !boxController.hasBorder()
+                  ? null
+                  : boxController.borderGradient != null
+                      ? GradientBoxBorder(
+                          gradient: boxController.borderGradient!,
+                          width: boxController.borderWidth?.toDouble() ??
+                              ThemeManager().getBorderThickness(context))
+                      : Border.all(
+                          color: boxController.borderColor ??
+                              ThemeManager().getBorderColor(context),
+                          width: boxController.borderWidth?.toDouble() ??
+                              ThemeManager().getBorderThickness(context)),
+              borderRadius: boxController.borderRadius?.getValue(),
+              boxShadow: !boxController.hasBoxShadow()
+                  ? null
+                  : <BoxShadow>[
+                      BoxShadow(
+                          color: boxController.shadowColor ??
+                              ThemeManager().getShadowColor(context),
+                          blurRadius: boxController.shadowRadius?.toDouble() ??
+                              ThemeManager().getShadowRadius(context),
+                          offset: boxController.shadowOffset ??
+                              ThemeManager().getShadowOffset(context),
+                          blurStyle: boxController.shadowStyle ??
+                              ThemeManager().getShadowStyle(context))
+                    ],
+            ),
+      child: backgroundImage != null
+          ? Stack(
+              children: [
+                backgroundImage,
```

Comment:
this will cause the Image to contribute to the Stack's size. We only want the main widget to determine the Stack's size, and the Image to act as the background. Use Position.fill()

Reply:
I updated it. Can you review it?